src/platform/monitoring/DowntimeNotification/actions/index.js

```diff
@@ -50,18 +55,42 @@ export function dismissDowntimeWarning(appTitle) {
 }
 
 export function getScheduledDowntime() {
-  return async dispatch => {
+  return async (dispatch, state) => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
     let data;
+
+    // create global downtime data if feature toggle is enabled and if
+    // current time is in downtime window
+    // default to empty array
+    const { downtimeStart, downtimeEnd } = scheduledDowntimeWindow;
+    const globalDowntimeData =
+      (toggleValues(state).vaGlobalDowntimeNotification &&
+        moment().isAfter(downtimeStart) &&
+        moment().isBefore(downtimeEnd) &&
+        createGlobalMaintenanceWindow({
+          startTime: downtimeStart,
+          endTime: downtimeEnd,
+        })) ||
+      [];
     try {
       const response = await apiRequest('/maintenance_windows/');
-      data = response.data;
+      if (!response.ok) {
```

Comment:
I wasn't able to verify that `apiRequest` returns this status. help is appreciated here.

Reply:
It [doesn't look like](https://github.com/department-of-veterans-affairs/vets-website/blob/b7433ca8ac919f2616351bef8492e97095b7b4f6/src/platform/utilities/api/index.js#L92-L98) `apiRequest` returns the `ok` prop, only the json data.  However, if the response _isn't_ ok then it should reject the promise which we could handle in a `.catch()`

Reply:
alrighty- thanks for the info.